provider/cmd/pulumi-resource-cloud-toolkit-aws/serverless/staticwebArgs.ts

```diff
@@ -0,0 +1,40 @@
+import * as aws from "@pulumi/aws";
+
+export interface StaticWebArgs {
+  domain?: string;
+  includeWWW?: boolean;
```

Comment:
Add comments to each one of the interface properties

Reply:
done